corva/middleware/loader.py

```diff
@@ -0,0 +1,13 @@
+from typing import Callable
+
+from corva.loader.base import BaseLoader
+from corva.models.base import BaseContext
+
+
+def loader_factory(*, loader: BaseLoader) -> Callable:
```

Comment:
why do we need this function to receive every argument before loader?

Reply:
also, this function is not used anywhere

Reply:
deleted keyword only argument

l`oader_factory` exists to parametrize `loader_`. e.g `Stream`, `Scheduled` and `Task` apps all have different loaders, but can use same middleware

it will be used in the next pr, where I add first app type

Reply:
can you please make PRs more completed? It gives a little insight into how functions are used. Reviewers cannot justify the implementation of something which is not used anywhere